lib/mimdb_web/templates/actor/form.html.eex

```diff
@@ -0,0 +1,19 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+
+  <%= label f, :name %>
+  <%= text_input f, :name %>
+  <%= error_tag f, :name %>
+
+  <%= label f, :birthdate %>
```

Comment:
Do we really only care about the year or do we actually want the exact date?

Reply:
<img width="295" alt="Screen Shot 2020-09-01 at 16 29 09" src="https://user-images.githubusercontent.com/214923/91897197-508afe00-ec70-11ea-9df5-e6a1e318c9ce.png">
@mratliff we care about Year/Month/Day, just are adding a range for Year's value.

Reply:
duh yea, sorry that was dull of me. You are right.